src/ConnectionAutomaton.cpp

```diff
@@ -241,15 +297,44 @@ void ConnectionAutomaton::cancel() {
 }
 /// @}
 
+void ConnectionAutomaton::scheduleKeepalive(const uint32_t keepAliveDelay) {
+  if (keepAliveDelay > 0) {
+    auto eventBase = folly::EventBaseManager::get()->getExistingEventBase();
```

Comment:
A question for discussion...
Grabbing current EventBase instance will work for all our platforms, but in general, should we wrap EventBase+runAfterDelay into a new interface called KeepAliveScheduler and initialize ConnectionAutomaton with an instance to the concrete scheduler? That way we can remove the dependency to EventBase from the core of ReactiveSocket..
What do you think?